### PR TITLE
open-mesh: Fix missing smoother files

### PR DESCRIPTION
**Pull request by M-Mueller**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----
Because of a typo in the cmake files, some files required for the Smoother were not installed, e.g. `#include <OpenMesh/Tools/Smoother/SmootherT.hh>` did not compile.

### CODE REVIEW

File: Formula/open-mesh.rb [hunk 1]
```diff
@@ -15,6 +15,11 @@ class OpenMesh < Formula
 
   depends_on "cmake" => :build
 
+  patch do
+    url "https://graphics.rwth-aachen.de:9000/OpenMesh/OpenMesh/commit/c5cfef87427a793268f9e012856872bbed958d92.diff"
```
Comment (MikeMcQuaid) [inline Formula/open-mesh.rb]: Can you please submit this patch to the upstream developers of this project and add a link to the upstream patch submission (if this is not an upstream commit, it's not clear) and regardless add an explanation of why the patch is needed in a comment in the formula file. Thanks!

**Review by MikeMcQuaid — CHANGES_REQUESTED** [review]

**Comment by M-Mueller** [comment]
I added the comment. The patch is from the upstream repo and integrated in OpenMesh 6.0. I think the project was moved from svn to git, maybe the url in the formula should be updated as well?

**Comment by dunn** [comment]
> I think the project was moved from svn to git, maybe the url in the formula should be updated as well?

Yes, please.  Thanks!

**Review by MikeMcQuaid — APPROVED** [review]

**Comment by MikeMcQuaid** [comment]
Thanks for your contribution to Homebrew! Without people like you submitting PRs we couldn't run this project. You rock!

**Comment by M-Mueller** [comment]
Thank you too, for making contributing so easy!

**Pull request closed without merging**